### DISCUSSION TITLE
Discord or Discourse forum?

### DISCUSSION THREAD

**Discussion by arladmin** [ToolJet/ToolJet#5433] [Ideas]
Any plans/interest in starting a community forum on either of these two popular platforms?

**Comment by withshubh** [comment]
Hey @arladmin - We already have an active **Community Slack** you can join it [here](https://tooljet.com/slack).

To answer your question - we don't plan to move our community from Slack (at least till we start receiving more requests). Our community has been operating smoothly on Slack since the beginning and till now we haven't found any major reason for moving our community to Discord. 

 Is there any specific reason for asking this? 🤔

**Reply by russell-hutson-jp** [reply]
Is this Slack link still supported?

**Reply by withshubh** [reply]
Yes!
